File: desafios/desafio_1/primos.js
function is_prime(n){
    if(n <= 1){
        return false;
    } else if (n == 2){
        return true;
    }else if (n%2 == 0){
        return false;
    }else{
        for (let i = 2; i < n; i++) {
            if(n%i == 0){
                return false;
            }         
        }
        return true;
    }
}

let count = 0;
let num = 0;
let primes = [];
while(count < 10){
    num++;
    if(is_prime(num)){
        primes.push(num);
        count++;
    }
}
console.log(primes);

File: desafios/desafio_1/primos.py
def is_prime(n):
    if n <= 1:
        return False
    elif n == 2:
        return True
    elif n % 2 == 0:
        return False
    else:
        for i in range(2,n):
            if(n%i==0):
                return False
        return True

if __name__ == '__main__':
    count = 0
    number = 0
    primes = []
    while count < 10:
        number+=1
        if is_prime(number):
            primes.append(number)
            count+=1
    print(primes)
